pkg/kp/statusstore/podstatus/status.go

```diff
@@ -23,6 +24,10 @@ const (
 	Pending  ServiceState = "pending"
 	Running  ServiceState = "running"
 	Finished ServiceState = "finished"
+
+	PodLaunched PodState = "launched"
```

Comment:
wow, that would be a heck of a bug! It's probably most safe to prefix them all with "pod_"